package.json

```diff
@@ -29,11 +29,13 @@
   },
   "dependencies": {
     "@hapi/joi": "^15.1.0",
+    "@types/helmet": "^0.0.45",
+    "@types/morgan": "^1.7.37",
```

Comment:
we aren't suppose to package types into production build unless it's a library that exposes that type

Reply:
argh, mistakenly went into prod depenencies by constant removing and adding